Cargo.toml

```diff
@@ -114,6 +115,10 @@ polkadot-cli = {git = 'https://github.com/paritytech/polkadot', branch = 'rococo
 node-runtime = { package = "centrifuge-chain-runtime", path = "runtime" }
 node-primitives = { branch = "rococo-v1", git = "https://github.com/paritytech/substrate.git" }
 
+# local dev dependencies
+cf-reward = { path = "pallets/cf-reward" }
+crowdloan-claim = { path = "pallets/crowdloan-claim" }
+
```

Comment:
As this rewarding process is meant for crowdloan campaigns, I would suggest to rename the dependency as `crowdloan-reward` and the pallet's folder renamed `pallets/crowdloan-reward`.